packages/manager/apps/dedicated/client/app/ip/ip-range.service.js

```diff
@@ -35,11 +35,10 @@ angular.module('Module.ip.services').service('IpRange', function IpRange() {
   function getStrCopy(str, _copies) {
     const copies = _copies > 0 ? _copies : 1;
 
-    return times(copies + 1, str).join('');
+    return times(copies, () => str).join('');
   }
 
-  function convertDecToBase(dec, base, _length, _padding) {
-    const padding = _padding || '0';
+  function convertDecToBase(dec, base, _length, padding = 0) {
```

Comment:
```suggestion
  function convertDecToBase(dec, base, _length, padding = '0') {
```
according to the previous code 😄